doc/about.rst

```diff
@@ -2,22 +2,85 @@
 About Secure XGBoost
 ####################
 
-**Secure XGBoost** enables collaborative `gradient boosting <https://en.wikipedia.org/wiki/Gradient_boosting>`_ in the multiparty setting, leveraging `hardware enclaves <https://inst.eecs.berkeley.edu/~cs261/fa18/slides/Hardware_Enclaves.pdf>`_ and `data oblivious algorithms <https://www.usenix.org/system/files/conference/usenixsecurity16/sec16_paper_ohrimenko.pdf>`_ to securely perform this joint computation. In particular, multiple parties can collaborate to compute a joint model on their sensitive data without revealing the contents of the data. 
 
-For example, one can imagine a group of banks that wants to jointly train an anti-money laundering machine learning model. This would require each bank to share its customer data. However, legal policies and privacy concerns prevent banks from sharing customer data in plaintext -- they should not revealto another bank, for example, the amount in a customer's savings account or a customer's recent transactions. Therefore, banks must jointly compute on all their sensitive data without explicitly sharing the data itself. This scenario can be generalized to a wide number of other industries in which parties may not want to share data because of data privacy laws or business competition: hospitals developing a disease diagnosis model, telecom companies predicting link failures, etc. 
+Motivation
+**********
 
-.. image:: images/coopetition.png
-   :scale: 80%
-   :alt: Coopetition
+Training a machine learning model requires a large quantity of high-quality data. One way to achieve this is to combine data from many different data organizations or data owners. But data owners are often unwilling to share their data with each other due to privacy concerns, which can stem from business competition, or be a matter of regulatory compliance.
+
+The question is: how can we mitigate such privacy concerns?
+
+Secure collaborative learning enables many data owners to **build robust models on their collective data, but without revealing their data to each other**. Banks can collaborate on anti-money laundering efforts while keeping their customer data private. Healthcare institutions can pool their patient data together and collaborate on medical studies. The possibilities are vast and promising.
+
+
+Secure XGBoost
+**************
+
+In a nutshell, multiple clients (or data owners) can use Secure XGBoost to collaboratively train an XGBoost model on their collective data in a cloud environment, while preserving the privacy of their individual data. Though we focus on collaborative learning in the rest of this document, Secure XGBoost also supports a single party who simply wants to outsource computation to the cloud but who doesn't want to transfer data to the cloud in plaintext. 
```

Comment:
Remove "Though we focus on collaborative learning in the rest of this document,"